tests/locale/keymap_or_locale.pm

```diff
@@ -18,7 +18,14 @@ use utils;
 sub verify_default_keymap_textmode {
     my ($test_string, $tag, %tty) = @_;
 
-    defined($tty{console}) ? select_console($tty{console}) : send_key('alt-f3');
+    if (defined($tty{console})) {
+        select_console($tty{console});
+    }
+    else {
+        send_key('alt-f3');
+        wait_still_screen;
```

Comment:
I found in another PR that `assert_screen 'linux-login';` can be used - it might be more reliable than `wait_still_screen`